ci/run-tests.ps1

```diff
@@ -1,11 +1,36 @@
 param(
     [string] $unreal_editor_path,
     [string] $uproject_path,
-    [string] $output_dir,
+    [string] $test_repo_path,
     [string] $log_file_path,
     [string] $test_repo_map
 )
 
+# Generate schema and snapshots
+Echo "Generating snapshot and schema for testing project"
```

Comment:
`Echo` in powershell is actually just an alias for `Write-Host` 😉

Reply:
Updated!